core/gather/gatherers/link-elements.js

```diff
@@ -70,6 +70,8 @@ function getLinkElementsInDOM() {
       crossOrigin: link.crossOrigin,
       hrefRaw,
       source,
+      // @ts-expect-error - Not in tsc types yet.
+      fetchPriority: link.fetchPriority,
```

Comment:
Is it possible to add an internal type declaration for this?

Reply:
> Is it possible to add an internal type declaration for this?

Yeah, it felt like overkill because it's [part of the html spec](https://html.spec.whatwg.org/multipage/semantics.html#dom-link-fetchpriority) and so it will eventually get rolled up into the tsc types, but OTOH `@ts-expect-error` is pretty heavy handed. Added to the DOM types in the somewhat misleadingly named `node.d.ts`